presto-product-tests/conf/docker/common/kerberos.yml

```diff
@@ -13,4 +13,4 @@ services:
   application-runner:
     image: 'teradatalabs/cdh5-hive-kerberized:${DOCKER_IMAGES_VERSION}'
     volumes:
-      - ../../../conf/tempto/tempto-configuration-for-docker-kerberos.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
+    - ${TEMPTO_CONFIG_KERBEROS_YAML}:/docker/volumes/tempto/tempto-configuration-local.yaml
```

Comment:
I tried that at first, but could not find a way to duplicate the original default behavior, where using run_on_docker.sh with a kerberos profile would run the tests using the open source JDBC driver without the user having to make any additional config or environment changes. 

If maintaining that behavior is not a requirement, then I agree a single environment variable in standard.yml will be better.